packages/forms-web-app/src/views/register/type-of-party.njk

```diff
@@ -0,0 +1,72 @@
+{% extends "layouts/main.njk" %}
```

Comment:
@pk2453 @AnthonyJWalsh Is it acceptable for the page **header** to _look_ different from design  ? Shall we raise additional ticket ( as this is something generic for the whole app as opposed to being specific to this ticket ) ? 
![VS](https://user-images.githubusercontent.com/91668344/137354364-40932179-3af3-4c36-9b17-32d7d8b1417a.png)